packages/expo-av/android/build.gradle

```diff
@@ -79,9 +112,27 @@ if (new File(rootProject.projectDir.parentFile, 'package.json').exists()) {
 }
 
 dependencies {
+  // noinspection GradleDynamicVersion
+  implementation 'com.facebook.react:react-native:+'
   unimodule "unimodules-core"
   unimodule 'expo-modules-core'
 
+  compileOnly 'com.facebook.fbjni:fbjni:0.2.2'
+  extractHeaders 'com.facebook.fbjni:fbjni:0.2.2:headers'
+  extractJNI 'com.facebook.fbjni:fbjni:0.2.2'
```

Comment:
I'm not an expert, but wouldn't those dependencies collide if you include expo-av into any bare/react-native project?

Reply:
I don't know for sure, do you know who might be able to answer that? How can I test this?

Reply:
Or is there another way you're aware of that might work better than extracting the fbjni lib?